archeo/style.css

```
@@ -179,4 +180,4 @@ body > .is-root-container > .wp-block-template-part > .wp-block-cover,
 .wp-block-post-comments input[type=submit].wp-block-button__link {
 	background-color: var(--wp--preset--color--background);
 	color: var(--wp--preset--color--foreground);
```

While we are here, should we update the font-family of the button?

```suggestion
	color: var(--wp--preset--color--foreground);
	font-family: var(--wp--preset--font-family--chivo);
```

---

Oh good spot. Could we do this via theme.json rather than CSS?